### DISCUSSION TITLE
Chainlink Spring Hackathon 2022 - Winners (so far)

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#1538] [General]
[medium article](https://n4n0b1t3.medium.com/chainlink-spring-hackathon-2022-winners-c43772808de4)

# The Grand Prize
## VMTree
https://devpost.com/software/chainlink-vmt
$???

# Top projects

### Nusic
https://devpost.com/software/nusic-layer-1-for-music
$25000
Layer 1 for Music

### Defi For The People
https://devpost.com/software/defi-for-people
$25000

### Chainlink Relayers
[Chainlink Relayers](https://devpost.com/software/chainlink-relayers)
$20000

### HapticDao
https://devpost.com/software/hapticdao
$20000

### 0xAudition
https://devpost.com/software/0xaudition
$20000

### Node Ops
https://devpost.com/software/node-ops
$10000

### Polynodes
https://devpost.com/software/polynodes
$7500

### Chainlink Job Specs Viz
https://devpost.com/software/chainlink-job-spec-viz
$5000

## Truflation
1. $5000 Inflotton
2. $3000 Plation, https://devpost.com/software/plation
3. $2000 I-bond, https://devpost.com/software/i-bond-with-trufflation

## Moralis
1. $7500 Taxme, https://devpost.com/software/taxme
2. $7500 Web3 Reputation Oracle, https://devpost.com/software/web3-reputation-oracle
3. Metaverse $? Shoot Ball, https://devpost.com/software/shoot-ball

## Filecoin
1. $10000 Gummys, https://devpost.com/software/gummys-57ysid (note: project is older than one year, only parts were added during the hackathon)
2. $7500 GeoNFT, https://devpost.com/software/geonft
3. $5000 Nusic https://devpost.com/software/nusic-layer-1-for-music

### Filecoin: Defi Magician
1. $5000 Meta4Swap, https://devpost.com/software/theo-daos
2. $2500 Speculate, https://devpost.com/software/speculate (private video?)

### Filecoin: Social Good
1. $5000 Endevr, https://devpost.com/software/ndver
2. $2500 Schain, https://devpost.com/software/schain-wtyzxf

### Filecoin: Storage Wizard
1. $10000 Gummys, see above
2. $7500 GeoNFT, see above
3. $5000 Nusic, see above

### Filecoin: On the Tools
1. $5000 Discharge, https://devpost.com/software/discharge
2. $2500 NFTStorageHelper, https://devpost.com/software/nftstorage-helper
3. $2500 Encryption Pinner, https://devpost.com/software/encryption-pinner

### Filecoin: Gaming, Metaverse, Daos & NFTs
1. $5000 IntoTheVerse, https://devpost.com/software/intotheverse-metaverse-land-sale-chainlink-moralis
2. $2500 Coollink, https://devpost.com/software/coollink-crypto-jumper

### Filecoin: IPFS It
1. $5000 Youtube-DL-dweb, https://devpost.com/software/youtube-dl-dweb

### Filecoin: Best Hackproject that we’ll use everyday
1. $5000 DPassword, https://devpost.com/software/dpassword-dapp-password-manager

**Comment by cromewar** [comment]
Nice info, all in one place. Loved it!
